packages/migration/migrations/hearth/20221115062056-update-status-in-complete-fields.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
+ * graphic logo are (registered/a) trademark(s) of Plan International.
+ */
+
+import { FieldType, InfluxDB } from 'influx'
+
+const INFLUX_HOST = process.env.INFLUX_HOST || 'localhost'
+const INFLUX_PORT = process.env.INFLUX_PORT || 8086
+const INFLUX_DB = process.env.INFLUX_DB || 'ocrvs'
+
+export const up = async (db) => {
+  const influx = new InfluxDB({
+    host: INFLUX_HOST,
+    database: INFLUX_DB,
+    port: INFLUX_PORT,
+    schema: [
+      {
+        measurement: 'in_complete_fields',
+        fields: {
+          regStatus: FieldType.STRING
+        },
+        tags: [
+          'compositionId',
+          'eventType',
+          'locationLevel5',
+          'locationLevel4',
+          'locationLevel3',
+          'locationLevel2',
+          'missingFieldGroupId',
+          'missingFieldId',
+          'missingFieldSectionId'
+        ]
+      }
+    ]
```

Comment:
@kayumuzzaman if you take a look at the schema here: https://github.com/opencrvs/opencrvs-core/blob/edd2e47262066f592240b0b7c2ae42126c28a54d/packages/metrics/src/influxdb/client.ts#L88-L104, `regStatus` is actually a tag and the only field should be a `compositionId`

Reply:
fixed